src/main/java/frc/robot/auton/RotatePID.java

```diff
@@ -15,14 +15,14 @@ public RotatePID(double relativeTurnDegree) {
             // This should return the measurement
             () -> Robot.getGyroscope().getAngle(),
             // This should return the setpoint (can also be a constant)
-            () -> Robot.getGyroscope().getAngle() + relativeTurnDegree,
+            Robot.getGyroscope().getAngle() + relativeTurnDegree,
```

Comment:
This was causing the setpoint to change every frame, which made this impossible to finish.